### DISCUSSION TITLE
Will Playground be available for self-hosted?

### DISCUSSION THREAD

**Discussion by sabatesduran** [langfuse/langfuse#2073] [Support] [answered]
I noticed that Playground is currently available only as a cloud service. Is there any plan to make it available for self-hosting in the future?

Thanks

**Comment by marcklingen** [comment]
Yes, it is currently in public beta and we want to release it as part of langfuse v3 for self-hosting as well

**Reply by secsilm** [reply]
Is there a planned date for v3?

**Reply by marcklingen** [reply]
June, no clear date as we want to get the rollout/migration right for all users

**Comment by uahmad235** [comment]
Any updates on this? @marcklingen

**Reply by marcklingen** [reply]
Yes, the playground is available for self-hosting as part of the Langfuse Enterprise Edition today. Please reach out in case you are interested: enterprise@langfuse.com

Context: We plan to keep all of the core features fully open source under a permissive license and will add some add-on features (such as the playground) and some security features relevant only in larger organizations as part of the Enterprise Edition. Learn more about this here: https://langfuse.com/docs/deployment/feature-overview

**Comment by martin-santiago** [comment]
Hi!, any updates on this? I've created a railway instance for langfuse v3.54.0 and I don't have the playground either, thanks in advance

**Comment by clemra** [comment]
You can now request a trial license for both Pro and EE versions to access these self-hosted: https://forms.gle/cXZuQZLmzJp8yd9k7

**Comment by marctorsoc** [comment, marked as answer]
Just updated to 3.75.4 and Playground is available!

**Reply by marcklingen** [reply]
Yes. Recently fixed this and open sourced all the previous “Pro” features. Learn more: https://langfuse.com/blog/2025-06-04-open-sourcing-langfuse-product
